django_admin_index/templatetags/django_admin_index.py

```diff
@@ -32,3 +32,8 @@ def dashboard_app_list(context):
 @register.simple_tag()
 def admin_index_settings():
     return settings.as_dict()
+
+
+@register.simple_tag
+def display_dropdown_menu(request):
+    return settings.DISPLAY_DROP_DOWN_MENU_CONDITION_FUNCTION(request)
```

Comment:
The setting should be a string value, something like `"django_admin_index.utils.should_display_dropdown_menu"`.

You can then import this function using `django.utils.module_loading.import_string` and invoke it:

```suggestion
def display_dropdown_menu(request):
    func = import_string(settings.DISPLAY_DROP_DOWN_MENU_CONDITION_FUNCTION)
    return func(request)
```

This makes it possible to pass the function to use as an environment variable.